### PR TITLE
Add suffixes to typos from "cl" to "col"

### PR DESCRIPTION
**Pull request by int-y1**
`codded` and `codding` are in aspell

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10517,9 +10517,27 @@ ckecksum->checksum
...
 claaes->classes
 claculate->calculate
+claculated->calculated
+claculates->calculates
+claculating->calculating
 claculation->calculation
+claculations->calculations
+claculator->calculator
+claculators->calculators
+claen->clean
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:10527]: Any occurrences of `clan`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10551,38 +10571,81 @@ classe->class, classes,
...
+classsify->classify
+classsifying->classifying
+classsroom->classroom
+classsrooms->classrooms
 clatified->clarified
 claus->clause
 clcoksource->clocksource
 clcosed->closed
-clea->clean
+clea->clean, clear,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:10614]: Perhaps `cleat` or `clew`, or even `plea`, although such typos must be rather rare.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10551,38 +10571,81 @@ classe->class, classes,
...
+cleaed->cleaned, cleared,
 cleaer->clear, clearer, cleaner,
 cleaered->cleared
+cleaering->clearing
+cleaers->clears, cleaners,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:10619]: `cleavers` too (also singular `cleaver`)?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -10551,38 +10571,81 @@ classe->class, classes,
...
 cleaing->cleaning, clearing,
 clealy->clearly, cleanly,
 cleancacne->cleancache
 cleand->cleaned, cleans, clean,
+cleandar->calendar
+cleandars->calendars
+cleanded->cleaned
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:10626]: Take into account `cleanse` in typos and fixes, in addition to `clean`?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -10816,14 +10931,23 @@ coctail->cocktail
 cocument->document
 cocumentation->documentation
 cocuments->documents
+cocurrency->concurrency
+cocurrent->concurrent
+cocurrently->concurrently
 codde->code, coded, coddle,
+codder->coder
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:10938]: Perhaps `coddler`? Probably rare, but that would be consistent with `codded->coded, coddled,`.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -10965,6 +11109,7 @@ collecions->collections
 colleciton->collection
 collecitons->collections
 collectin->collection
+collectins->collections
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:11112]: Perhaps add `collective`/`collectives` and `collector`/`collectors` typos wherever they're missing?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -11030,7 +11182,13 @@ coloums->columns
 colourfull->colourful, colourfully,
 colourpsace->colourspace
 colourpsaces->colourspaces
+colse->close
 colsed->closed
+colsely->closely
+colser->closer
+colses->closes
+colsest->closest
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:11190]: `colsest->closest, coldest,`

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -11030,7 +11182,13 @@ coloums->columns
 colourfull->colourful, colourfully,
 colourpsace->colourspace
 colourpsaces->colourspaces
+colse->close
 colsed->closed
+colsely->closely
+colser->closer
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:11188]: `colser->closer, colder,`

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -11030,7 +11182,13 @@ coloums->columns
...
+colses->closes
+colsest->closest
+colsing->closing
 colud->could, cloud,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:11192]: Perhaps `collude` too?

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:10614]: I will add `cleat`. `clew` and `plea` are too rare.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:10527]: I couldn't find `clan`.

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:11112]: I will add these to the `collet->collect` root word.

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:11192]: I can't find `colud->collude`.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**